backend/dataall/core/organizations/db/organization_repositories.py

```diff
@@ -141,3 +141,16 @@ def find_organization_membership(session, uri, groups) -> int:
             .count()
         )
         return groups >= 1
+
+    @staticmethod
+    def is_group_invited(session, uri, group) -> bool:
```

Comment:
It is almost equal to `find_organization_membership`, do you think we could re-use it instead of creating a new function?

Reply:
And we also have `find_group_membership` that is also super similar to the other 2

Reply:
Refactored everything to use only `find_group_membership`

Reply:
Love it!